README.md

```diff
@@ -82,3 +82,52 @@ the exact (byte-to-byte) same binaries which are published to https://releases.h
 ### Why doesn't HashiCorp maintain formulae in the `homebrew-core` (main tap)?
 
 Homebrew's core team prefers to keep `homebrew-core` as community maintained and built from source to maintain consistency across vendors in terms of expectations around contents and updates. See [relevant discussion](https://discourse.brew.sh/t/maintenance-of-formulas-by-vendor/7649) for more information.
+
+## Contributing
```

Comment:
FYI GitHub has a way of highlighting the contribution guidelines when they are in a specific place in the repo, so you may want to consider moving it there and just linking from here

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file

Reply:
Ah right, CONTRIBUTING.md. I'll make that change -- thank you!